src/components/common/Notification/NotificationBar.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div
+    :class="[
+      'noti',
+      alertType,
+      'animate__animated',
+      show && 'show',
+      show ? 'animate__fadeInRight' : 'animate__fadeOutRight',
+    ]"
+  >
+    <div class="noti-content" @mouseover="showCloseBtn = true">
+      <div class="row--close">
+        <div class="column--title">
+          <span v-if="isSuccessType" class="icon--check">
+            <astar-icon-circle-check size="20" />
+          </span>
+          <span v-else class="icon--check">
+            <astar-icon-warning size="20" />
+          </span>
+          {{ alertTypeTitle }}
+        </div>
+        <div v-if="showCloseBtn" class="column--close" @click="close">
+          <astar-icon-close />
+        </div>
+      </div>
+      <div>
+        <div class="message">{{ alertMsg }}</div>
+        <astar-button v-if="isSuccessType" class="btn--check" @click="goToSubscan"
+          >Check your transactions</astar-button
+        >
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { AlertType } from 'src/store/general/state';
+import { useNetworkInfo } from 'src/hooks';
+import { endpointKey } from 'src/config/chainEndpoints';
+import { defineComponent, toRefs, PropType, computed, ref } from 'vue';
+
+export default defineComponent({
+  name: 'NotificationBar',
+  props: {
+    show: {
+      type: Boolean,
+      default: false,
+    },
+    alertType: {
+      type: String as PropType<AlertType>,
+      required: true,
+    },
+    alertMsg: {
+      type: String,
+      required: true,
+    },
+    txHash: {
+      type: String,
+      default: null,
+    },
+  },
+  emits: ['close'],
+  setup(props, { emit }) {
+    const close = () => {
+      emit('close');
+    };
+
+    const alertTypeTitle = computed(() => {
+      if (props.alertType === AlertType.Success) {
+        return 'Success';
```

Comment:
Is it possible to translate?

Reply:
Thanks, fixed in [here](https://github.com/AstarNetwork/astar-apps/pull/644/commits/f6b8488d36e8cca415749aca1f3ecee416ee7320) . regarding transaction button, if no txHash, the button would not be visible now. I'll link proper txHash on the refactor PR.